portfolio.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Joseph Farmer</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <h1 class="header__title">Joseph Farmer</h1>
+        <nav class="navigate">
+            <ul class="header__menu">
+                <li class="header__menu-item">
+                    <a href="index.html">home</a>
+                </li>
+                <li class="header__menu-item">
+                    <a href="portfolio.html">Projects</a>
+                </li>
+                <li class="header__menu-item">
+                    <a href="resume.html">Resume</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main class="main">
+        <h2 class="projects__header"> Projects</h2>
+        <div class="projects">
+            <h3 class="projects__header__1"> Garden Supply Store</h3>
```

Comment:
I think you could abstract this by using classes like `project__header` without specifying 1 or 2. They should have similar CSS.

Reply:
To note, if they had different CSS, you could still have something like `<h3 class="project__header project__header--1">` and put the CSS that was the same on the `project__header` class and the differing CSS on the `project__header--1` class.